components/calendar/calendar.js

```diff
@@ -134,6 +134,9 @@ class Calendar extends LocalizeStaticMixin(LitElement) {
 			summary: { type: String },
 			_dialog: { type: Boolean },
 			_focusDate: { type: Object },
+			_initialMonth: { type: Boolean },
+			_nextMonthNav: { type: Boolean },
+			_prevMonthNav: { type: Boolean },
```

Comment:
Only one of these can the `true` at once I think ya? So could it just be a single property that gets set to either `initial`, `next` or `prev`?